vite.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig(() => ({
   plugins: [react()],
   test: {
     environment: "happy-dom",
-    include: ["src/**/*.spec.jsx"],
+    include: ["src/**/*.spec.js(x)?"],
```

Comment:
being pedantic here but the `()` are superfluous as the `?` always applies to the thing immediately before it, and when it's a single char we don't need to put it in a group

Reply:
I'm not sure that's true in a glob pattern? `src/**/*.spec.jsx?` doesn't match any test files, so I've left it as is.  Good to know for other regex tho!